test/cmake/seqan3_require_benchmark.cmake

```diff
@@ -48,3 +46,29 @@ macro (seqan3_require_benchmark)
 
     unset (gbenchmark_path)
 endmacro ()
+
+macro (seqan3_require_benchmark)
+    enable_testing ()
+
+    set (gbenchmark_git_tag "v1.5.0")
+
+    if (NOT CMAKE_VERSION VERSION_LESS 3.14)
```

Comment:
Does this `if / else` exist because the fetch does not work under version 3.14?

Reply:
perfect score :100:!